go/grpcweb/wrapper_test.go

```diff
@@ -340,14 +355,34 @@ func serializeProtoMessages(messages []proto.Message) [][]byte {
 	return out
 }
 
-func readHeadersFromBytes(dataBytes []byte) http.Header {
+func readHeadersFromBytes(t *testing.T, dataBytes []byte) http.Header {
```

Comment:
If this method is only used for reading trailers now (I can't see it used anywhere else), lets rename it `readTrailersFromBytes` and return `grpcweb.Trailer` instead?

Reply:
Good idea, I'll rename it.